### PR TITLE
Cidr 1374

### PR DESCRIPTION
**Pull request by KJStick**

### CODE REVIEW

File: wxt_cidrbot/dynamo_api_handler.py [hunk 1]
```diff
@@ -54,14 +54,47 @@ def __init__(self):
...
-        self.table = self.dynamodb.Table(self.db_room_name)
+        """
+        if not already done, creates a dynamodb session and switches the current table to the room table
+        
+        return: None
+        """
+        if self.dynamodb is None:
+            self.dynamodb = boto3.resource('dynamodb')
+        if self.current_table_name != self.db_room_name:
+            self.table = self.dynamodb.Table(self.db_room_name)
```
Comment (jlothian) [inline wxt_cidrbot/dynamo_api_handler.py:74]: if you are using different tables for different things, they should each likely have their own variable or entry in a dict, something like that

File: wxt_cidrbot/dynamo_api_handler.py [hunk 2]
```diff
@@ -54,14 +54,47 @@ def __init__(self):
...
+    def get_dynamo_installation_table(self):
+        """
+        if not already done, creates a dynamodb session and switches the current table to the installation table
+        
+        return: None
+        """
+        if self.dynamodb is None:
+            self.dynamodb = boto3.resource('dynamodb')
+        if self.current_table_name != self.db_install_name:
+            self.table = self.dynamodb.Table(self.db_install_name)
```
Comment (jlothian) [inline wxt_cidrbot/dynamo_api_handler.py:86]: see above

**Review by jlothian — CHANGES_REQUESTED** [review]

**Review by ppajersk — APPROVED** [review]
Lgtm

**Review by jlothian — APPROVED** [review]
Looks good.

**Pull request merged**